_includes/configuration/advtable_value_series.md

```diff
@@ -0,0 +1,91 @@
+## `advtable_value_series`
+{{ site.requires_5_9v }}
+
+The `advtable_value_series` option configures value series for populating cells in a table. This option can be used to create row identifiers.
```

Comment:
```suggestion
The `advtable_value_series` option configures a series of values for populating cells in a table. This option can be used to create row identifiers.
```
Maybe?

Reply:
I think my only concern with that is that it could sound like only one value series can be created. Maybe being more explicit would be better?
```suggestion
The `advtable_value_series` option configures one or more value series for populating cells in a table. This option can be used to create row identifiers.
```

Reply:
```suggestion
The `advtable_value_series` option configures one or more series of values for populating cells in a table. This option can be used to create row identifiers.
```
Split the difference?